addons/mass_mailing/static/src/js/mailing_mailing_view_form_full_width.js

```diff
@@ -59,7 +59,8 @@ const MassMailingFullWidthFormController = FormController.extend({
         } else {
             const ref = $iframeDoc.find('#iframe_target')[0];
             if (ref) {
-                this.$iframe.height(Math.max(ref.scrollHeight + VERTICAL_OFFSET, minHeight));
+                const isFullscreen = this._isFullScreen();
+                this.$iframe.css({ height: isFullscreen ? $(window).height() : Math.max(ref.scrollHeight + VERTICAL_OFFSET, minHeight) });
```

Comment:
I don't see the added value of creating a local `isFullscreen` variable only to use it once right after creation. Also, I suggest to rewrite the line for readability. Therefore, the resulting code would be:
```js
this.$iframe.css({
    height: isFullscreen
        ? $(window).height()
        : Math.max(ref.scrollHeight + VERTICAL_OFFSET, minHeight),
});
```

Reply:
Also, could you please try to remove the line at https://github.com/odoo/odoo/blob/a1f2826e690224463d7688582623203bb27e6849/addons/mass_mailing/static/src/js/mailing_mailing_view_form_full_width.js#L80 and see if it still works ? I think it is not needed anymore thanks to your fix.